### PR TITLE
Improve the Linux setup documentation

### PR DESCRIPTION
**Pull request by steverabino**
As a newbie to Linux myself, I found it pretty difficult to set up, and ended up doing more things than I needed to do.

This change attempts to split out the Linux setup steps into different pieces of functionality so users can decide if they need that function, or live without it. It also adds instructions of how to set up a daemon, which comes free with Mac.

I've tried to not be too prescriptive with these steps as I'm aware that I have limited exposure to different Linux distros, so would be happy to change if there's anything here that's deemed "too ubuntu"!

Mostly following advice on https://github.com/puma/puma-dev/issues/155

### CODE REVIEW

**Comment by nateberkopec** [comment]
Would love a review from anyone using puma-dev on Linux.

File: README.md [hunk 1]
```diff
@@ -78,26 +78,65 @@ When puma-dev is installed as a user agent (the default mode), it will log outpu
...
+------
 
 ## Linux Support
 
-Puma-dev supports Linux but requires additional installation to make all the features work. For example, `-install` and `-setup` flags for Linux are not provided, puma-dev root CA is generated but not installed or trusted.
+Puma-dev supports Linux but requires the following additional installation steps to be followed to make all the features work (`-install` and `-setup` flags for Linux are not provided):
+
+### puma-dev root CA
+
+The puma-dev root CA is generated (in `~/.puma-dev-ssl/`), but you will need to install and trust this as a Certificate Authority.
```
Comment (YoranBrondsema) [inline README.md]: For Chrome:

1. `Settings`
2. `Advanced` --> `Privacy and security`
3. `Manage certificates`
4. Open tab `Authorities`
5. `Import`
6. Select file `~/.puma-dev-ssl/cert.pem`

**Comment by YoranBrondsema** [comment]
I just followed the steps and managed to set up `puma-dev` on Ubuntu 18.04. It works great and thanks to the system daemon, it also works between computer restarts.

I added a small comment on how to add the CA to Chrome.

Thanks a bunch @steverabino! Very timely PR.

**Comment by nateberkopec** [comment]
Let's address @YoranBrondsema's comment and then I'll merge. Thanks for the review Yoran!

File: README.md (hunk 1 above)
Comment (steverabino) [reply README.md]: I decided against putting specific instructions for this step:
* there are multiple ways to do this (adding to the operating system trust store vs. doing so in a specific browser), and the complexities around which browsers use/do not use the operating system store
* multiple browsers (should I add instructions for Firefox, Opera, etc)
* changes that will occur between browser versions.

I will add a bit more details about the options of using the operating system certificate trust store vs. adding directly to the browser which should give enough information to the user to be able to search online for information

**Pull request merged**